include/revng/Support/GraphAlgorithms.h

```diff
@@ -4,14 +4,20 @@
 // This file is distributed under the MIT License. See LICENSE.md for details.
 //
 
+#include <concepts>
 #include <set>
+#include <variant>
 #include <vector>
 
 #include "llvm/ADT/GraphTraits.h"
+#include "llvm/ADT/PostOrderIterator.h"
 #include "llvm/ADT/SCCIterator.h"
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/ADT/SmallPtrSet.h"
 
+#include "revng/ADT/Concepts.h"
+#include "revng/ADT/GenericGraph.h"
+#include "revng/ADT/STLExtras.h"
```

Comment:
Can you please double check which of these headers are not needed anymore? I bet `concepts` `variant` and most `revng/ADT` can be dropped.

Reply:
Will do!

Reply:
There's nothing to remove. Most of these are gone along with the commits I dropped.